python/lsst/validate/drp/plot.py

```diff
@@ -552,3 +552,62 @@ def plotAMx(amx, afx, filterName, amxSpecName='design', outputPrefix=""):
     plt.tight_layout()  # fix padding
     plt.savefig(plotPath, dpi=300)
     plt.close(fig)
+
+
+def plotTEx(tex, filterName, texSpecName='design', outputPrefix=''):
+    fig = plt.figure(figsize=(10, 6))
+    ax1 = fig.add_subplot(1, 1, 1)
+    # Plot correlation vs. radius
+    ax1.errorbar(tex.radius.value, tex.xip.value, yerr=tex.xip_err.value)
+    ax1.set_xscale('log')
+    ax1.set_xlabel('Separation (arcmin)', size=19)
+    ax1.set_ylabel('Median Residual Ellipticity Correlation', size=19)
+
+    # Overlay requirements level
+    texSpec = tex.metric.get_spec(texSpecName, filter_name=filterName)
+    texSpecLabel = '{tex.label} {specname}: {texSpec:.2g}'.format(
+        tex=tex,
+        texSpec=tex.metric.get_spec(texSpecName,
+                                    filter_name=filterName).quantity,
+        specname=texSpecName)
+    ax1.axhline(texSpec.quantity.value, 0, 1, linewidth=2, color='red',
+                label=texSpecLabel)
+
+    # Overlay measured KPM whether it passed or failed.
+    if tex.check_spec(texSpecName):
+        texStatus = 'passed'
+    else:
+        texStatus = 'failed'
+    texLabelTemplate = '{tex.label} measured: {tex.quantity:.2g} ({status})'
+    texLabel = texLabelTemplate.format(tex=tex, status=texStatus)
+
+    ax1.axhline(tex.quantity.value, 0, 1, linewidth=2, color='black',
+                label=texLabel)
+
+    titleTemplate = """
+        {metric} Residual PSF Ellipticity Correlation
+        {bin_range_operator:s} {D.value:.1f}{D.unit:latex}
+        """
+    title = titleTemplate.format(metric=tex.label,
+                                 bin_range_operator=tex.bin_range_operator,
+                                 D=tex.D)
+    ax1.set_title(title)
+    ax1.set_xlim(0.0, 20.0)
+    ax1.set_xlabel(
+        '{radius.label} ({radius.latex_unit})'.format(
+            radius=tex.extras['radius']))
+    ax1.set_ylabel('Correlation')
+
+    ax1.legend(loc='upper right', fontsize=16)
+
+    pathFormat = '{prefix}{metric}_D_{D:d}_{Dunits}.{ext}'
```

Comment:
This should probably combine the name (`metric` and beyond) with the prefix via `os.path.join()`

Reply:
Ah. A comment to that effect would be good (prefix suggests directory to me).